srml/staking/src/lib.rs

```diff
@@ -555,6 +555,22 @@ pub trait Trait: system::Trait {
 	type SessionInterface: self::SessionInterface<Self::AccountId>;
 }
 
+/// Mode of era-forcing.
+#[derive(Copy, Clone, PartialEq, Eq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
+pub enum Forcing {
+	/// Not forcing anything - just let whatever happen.
+	NotForcing,
+	/// Force a new era, then reset to `NotForcing` as soon as it is done.
+	ForceNew,
+	/// Avoid a new era indefinitely.
+	ForceNone,
```

Comment:
more idiomatic would be to have `Forcing::New` and `Forcing::Avoid`, with storage holding an `Option<Forcing>`

Reply:
(typical Rust style would call this `Force` as opposed to `Forcing`)

Reply:
yeah, didn't manage to make that work with the module JSON config stuff. let me try again though - might have been some other issue.

Reply:
yeah, genesis configs require `Option` items to be `Some` value.

Reply:
See branch `gav-disable-eras-alt`

Reply:
OK, that's a bit unfortunate. but no reason to block this PR on fixing the macro

Reply:
Regardless, the repetition of `Force` in the variant names is redundant and it would be a bit cleaner without